openapi.yaml

```diff
@@ -469,1250 +491,1393 @@ paths:
+      type: object
+      additionalProperties: false
+      properties:
+        suffix:
+          type: string
+          nullable: true
+          description: >
+            Used to uniquely identify a job record for persons who have multiple
+            jobs that share the same position number. For updating existing
+            jobs, this field is required.
+        positionNumber:
+          type: string
+          description: Identifies a position.
+          example: C12345
+        employmentType:
+          type: string
+          enum: *employmentTypeEnums
+          description: Type of employment.
+        effectiveDate:
+          type: string
+          format: date
+          description: The date this job assignment is effective.
+          example: '1970-01-21'
+        beginDate:
+          type: string
+          format: date
+          description: Begin date of job.
+          example: '1970-01-21'
+        endDate:
+          type: string
+          nullable: true
+          format: date
+          description: End date of job.
+          example: '1971-02-15'
+        accruesLeaveInd:
+          type: boolean
+          description: If true, the job accrues leave time.
+        contractBeginDate:
+          type: string
+          nullable: true
+          format: date
+          description: The date the contract begins.
+          example: '1970-01-21'
+        contractEndDate:
+          type: string
+          nullable: true
+          format: date
+          description: The date the contract ends.
+          example: '1971-02-21'
+        contractType:
+          type: object
+          nullable: true
+          properties:
+            code:
+              type: string
+              enum:
+                [P, S, O]
+              description: |
+                Contract type code/Id.
+                - `P` - Primary
+                - `S` - Secondary
+                - `O` - Overload
+        campus:
+          type: object
+          nullable: true
+          properties:
+            code:
+              type: string
+              description: ID of campus where this job will be performed.
+        status:
+          type: object
+          properties:
+            code:
+              type: string
+              description: Job status.
+              enum: *statusCodeEnums
+        jobDescription:
+          type: string
+          nullable: true
+          description: Description of job.
+        personnelChangeDate:
+          type: string
+          nullable: true
+          format: date
+          description: Date the last change was made to the job record.
+          example: '1970-01-01'
+        changeReason:
+          type: object
+          nullable: true
+          properties:
+            code:
+              type: string
+              description: The reason the last change was made to the job record.
+        fullTimeEquivalency:
+          type: number
+          nullable: true
```

Comment:
This can't be null. Can you check all the rest attributes and make sure they can or can't be null?

```SQL
SELECT COUNT(*) FROM TABLE_NAME WHERE COLUMN_NAME IS NULL;
```